geoip2/records.py

```diff
@@ -684,6 +684,19 @@ class Traits(Record):
 
       :type: unicode
 
+    .. attribute:: static_ip_score
+
+      An indicator of how static or dynamic an IP address is. The value ranges
+      from 0 to 99.99 with higher values meaning a greater static association.
+      For example, many IP addresses with a user_type of cellular have a
+      lifetime under one. Static Cable/DSL IPs typically have a lifetime above
+      thirty.
+
+      This indicator can be useful for deciding whether an IP address represents
+      the same user over time.
+
```

Comment:
Could we say that it's only available from GeoIP2 Precision Insights, like we do with user_count?

Reply:
Changed and also corrected 2018 to 2019